api/cluster.go

```diff
@@ -621,31 +643,22 @@ func queryPeers(ctx context.Context, peerGroups map[int32][]cluster.Node, name s
 
 			case <-tickChan:
 				// Check if it's time to speculate!
-				percentReceived := float64(len(receivedResponses)) / float64(len(peerGroups))
-				if percentReceived >= speculationThreshold {
+				ratioReceived := float64(len(receivedResponses)) / float64(len(peerGroups))
```

Comment:
To be completely pedantic, percentage *is* a ratio. I like it here since it implies a clear relationship with the `speculationThreshold` though it may be confusing since it is represented as `[0-1]` rather than `[0-100]`, but I don't think ratio makes that any clearer.

Reply:
seems like this can be argued both ways. taking the path of least resistance..